src/api/PDFDocument.ts

```diff
@@ -147,6 +149,42 @@ export default class PDFDocument {
       throwOnInvalidObject,
       capNumbers,
     ).parseDocument();
+
+    if (returnCopy) {
+      const pdfDoc = new PDFDocument(context, ignoreEncryption, updateMetadata);
+      const pdfCopy = await PDFDocument.create();
+      const contentPages = await pdfCopy.copyPages(
+        pdfDoc,
+        pdfDoc.getPageIndices(),
+      );
+
+      for (const page of contentPages) {
+        pdfCopy.addPage(page);
+      }
+      if (pdfDoc.getAuthor() !== undefined) {
+        pdfCopy.setAuthor(pdfDoc.getAuthor()!);
+      }
+      if (pdfDoc.getCreationDate() !== undefined) {
+        pdfCopy.setCreationDate(pdfDoc.getCreationDate()!);
+      }
+      if (pdfDoc.getCreator() !== undefined) {
+        pdfCopy.setCreator(pdfDoc.getCreator()!);
+      }
+      if (pdfDoc.getModificationDate() !== undefined) {
+        pdfCopy.setModificationDate(pdfDoc.getModificationDate()!);
+      }
+      if (pdfDoc.getProducer() !== undefined) {
+        pdfCopy.setProducer(pdfDoc.getProducer()!);
+      }
+      if (pdfDoc.getSubject() !== undefined) {
+        pdfCopy.setSubject(pdfDoc.getSubject()!);
+      }
+      if (pdfDoc.getTitle() !== undefined) pdfCopy.setTitle(pdfDoc.getTitle()!);
+      pdfCopy.defaultWordBreaks = pdfDoc.defaultWordBreaks;
+
+      return pdfCopy;
+    }
```

Comment:
These changes are related to, but still distinct from, loading a document. So I think this would fit better as a `.copy()` method on `PDFDocument`. Then, instead of:
```js
PDFDocument.load(..., { returnCopy: true })
```
we could do:
```js
PDFDocument.load(...).copy()
```

Reply:
Once you've done this, be sure to add a proper doc comment to the `.copy()` method